tests/test_wave_energy.py

```diff
@@ -528,12 +523,12 @@ def test_valuation_suffix(self):
                     os.path.join(args['workspace_dir'], 'output', table_path)))
 
     @staticmethod
-    def _assert_point_vectors_equal(a_path, b_path):
+    def _assert_point_vectors_equal(a_vector_path, b_vector_path):
         """Assert that two point geometries in the vectors are equal.
 
-        Parameters:
-            a_path (str): a path to an OGR vector.
-            b_path (str): a path to an OGR vector.
+        Args:
+            a_vector_path (str): a path to an OGR vector.
+            b_vector_path (str): a path to an OGR vector.

```

Comment:
Especially if we end up moving the other vector-testing method to a function in `utils`, I wonder if this assertion method could also be subsumed by it?  At the very least, it'd be another use case.

Reply:
This assertion method is gone.